tests/testcases/test_user.py

```diff
@@ -52,3 +54,13 @@ def test_get_user(self):
             del os.environ['SUDO_USER']
             os.environ['PKEXEC_UID'] = '1000'
             self.assertNotEqual(get_user(), 'root')
+
+    def test_get_home(self):
+        with mock.patch('pwd.getpwnam', return_value = None):
+            self.assertEqual(get_home('foo'), '/home/foo')
```

Comment:
Those two lines are obsolete then

Reply:
this actually fails. But we don't have a reason to believe it would also fail like that during runtime

Reply:
Mmm.. I expected this to fail (no more try and you're None is provided) but somehow tests run fine. I think it is probably using the package installed on my computer for the test or something like that.

Yes, this will not fail at runtime (we determined that when we removed the try block + fallback).

I'll remove this test.